openapi/auth-server.yaml

```diff
@@ -362,51 +500,43 @@ components:
       required:
         - redirect
         - finish
-    continue:
-      title: continue
-      type: object
-      description: 'If the AS determines that the request can be continued with additional requests, it responds with the continue field.'
-      properties:
-        access_token:
-          type: object
-          description: 'A unique access token for continuing the request, called the "continuation access token".'
-          required:
-            - value
-          properties:
-            value:
-              type: string
-        uri:
-          type: string
-          format: uri
-          description: The URI at which the client instance can make continuation requests.
-        wait:
-          type: integer
-          description: The amount of time in integer seconds the client instance MUST wait after receiving this request continuation response and calling the continuation URI.
-      required:
-        - access_token
-        - uri
-    access_token:
-      title: access_token
-      type: object
-      description: A single access token or set of access tokens that the client instance can use to call the RS on behalf of the RO.
+    interval:
+      title: Interval
+      type: string
+      description: '[ISO8601 repeating interval](https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals)'
+      examples:
+        - 'R11/2022-08-24T14:15:22Z/P1M'
+        - 'R/2017-03-01T13:00:00Z/2018-05-11T15:30:00Z'
+        - 'R-1/P1Y2M10DT2H30M/2022-05-11T15:30:00Z'
+    limits-outgoing:
+      title: limits-outgoing
+      description: Open Payments specific property that defines the limits under which outgoing payments can be created.
+      type: [object, 'null']
```

Comment:
```suggestion
```
Do we need it if its non-required already?
also:
<img width="548" alt="Screenshot 2023-03-10 at 13 55 04" src="https://user-images.githubusercontent.com/15069181/224321326-baab9b03-ecf2-4d05-b188-247a20b918ed.png">

Reply:
If it's non-required, I think we can just put `type: object`